composer.json

```diff
@@ -34,6 +35,7 @@
         "codeception/module-cli": "^2.0",
         "codeception/module-filesystem": "^3.0",
         "codeception/module-phpbrowser": "^3.0",
+        "laravel/framework": "^9.48 || ^10.0",
```

Comment:
this package is made to support both Laravel and Lumen. I'm not sure that require Laravel is a good idea.

Reply:
This is only require-dev deps.